pym/bob/audit.py

```diff
@@ -146,6 +148,7 @@ def load(self, data):
         self.__defines = data["meta"]
         self.__build = data["build"]
         self.__env = data["env"]
+        self.__metaEnv = data["metaEnv"]
```

Comment:
Use `data.get("metaEnv", {})` because the key might not exist due to the change above...